package.json

```diff
@@ -40,7 +40,7 @@
     "tmp": "^0.1.0",
     "unzipper": "^0.9.3",
     "update-notifier": "^2.5.0",
-    "yargs": "12.0.5"
+    "yargs": "^15.1.0"
```

Comment:
If I remember well, this will give us access to patch versions, anything in the range 15.1.[0-N]

Reply:
I'm inclined to hard-pin the versions (we can actually do this for the rest of dependencies too). More than once even a small project like "craft" was hit by weird breakages/inconsistencies introduced in minor/patch versions.

Reply:
Yeah, I know that pain. Wouldn't be surprised if a patch version would burn datacenters.

Reply:
Made it require the latest version only, `15.1.0`.

Lots of deps using version ranges, I'd leave hard-pinning them all to a follow up.